azext_iot/tests/central/json/x509_certificate.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
just generate a certificate using the certificate commands in test_utils.py, and then during tear down, check if the certificate is there and delete it if needed.

Reply:
Fixed.